services/api-server/src/simcore_service_api_server/utils/solver_job_models_converters.py

```diff
@@ -0,0 +1,278 @@
+"""
+    Helper functions to convert models used in
+    services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py
+"""
+import urllib.parse
+import uuid
+from datetime import datetime
+from functools import lru_cache
+from typing import Callable, Dict, Optional, Tuple
+
+from models_library.projects_nodes import InputID, InputTypes
+
+from ..models.api_resources import compose_resource_name
+from ..models.domain.projects import (
+    InputTypes,
+    NewProjectIn,
+    Node,
+    Project,
+    SimCoreFileLink,
+    StudyUI,
+)
+from ..models.schemas.files import File
+from ..models.schemas.jobs import ArgumentType, Job, JobInputs, JobStatus, TaskStates
+from ..models.schemas.solvers import Solver, SolverKeyId, VersionStr
+from ..modules.director_v2 import ComputationTaskOut
+
+# UTILS ------
+_BASE_UUID = uuid.UUID("231e13db-6bc6-4f64-ba56-2ee2c73b9f09")
+
+
+@lru_cache()
+def compose_uuid_from(*values) -> str:
+    composition = "/".join(map(str, values))
+    new_uuid = uuid.uuid5(_BASE_UUID, composition)
+    return str(new_uuid)
+
+
+def format_dt(snapshot: datetime) -> str:
+    return "{}Z".format(snapshot.isoformat(timespec="milliseconds"))
+
+
+def now_str() -> str:
+    return format_dt(datetime.utcnow())
```

Comment:
maybe merge them in one function called `now_str_utc`

Reply:
I use this formatter function somewhere else and I we want all time-stamps in UTC in the entire backend.
In any case, all these functions will be moved to a new ``servicelib`` that we need to ensemble for fast-api like applications